app/views/widgets/my_module/_widget_samples.html.erb

```diff
@@ -1,2 +1,29 @@
 <div class="widget-content">
+  <div class="samples-table">
+    <% organization = widget.my_module.experiment.project.organization %>
+    <table id="samples" class="table"
+      data-organization-id="<%= organization.id %>"
+      data-source="<%= samples_index_my_module_path(widget.my_module, format: :json) %>"
+      data-num-columns=7
+      data-new-sample="<%= new_organization_sample_path(organization_id: organization) %>"
+      data-create-sample="<%= organization_samples_path(organization_id: organization) %>"
+      >
```

Comment:
Soo, the `data-organization-id`, `data-new-sample` and `data-create-sample` attributes are unneccesary. Also, the `data-num-column` can be skipped and setup in JS.